content/telegraf/v1/input-plugins/dpdk/_index.md

```diff
@@ -243,7 +244,20 @@ measurements.
 
 The DPDK socket accepts `command,params` requests and returns metric data in
 JSON format. All metrics from DPDK socket become flattened using Telegraf's
-JSON Flattener, and a set of tags that identify
+JSON Flattener and exposed as fields.  If DPDK
+response contains no information (is empty or is null) then such response will
+be discarded.
+
+> **NOTE:** Since DPDK allows registering custom metrics in its telemetry
+> framework the JSON response from DPDK may contain various sets of metrics.
+> While metrics from `/ethdev/stats` should be most stable, the `/ethdev/xstats`
```

Comment:
```suggestion
> While metrics from `/ethdev/stats` should be mostly stable, the `/ethdev/xstats`
```
Nit

Reply:
We should fix this also in the Telegraf repo.